docs/pages/includes/plugins/identity-export.mdx

```diff
@@ -53,6 +52,36 @@ This command should result in three PEM-encoded files: `auth.crt`,
 respectively). Later, you will configure the plugin to use these credentials to
 connect to the Auth Service directly.
 
+</TabItem>
+<TabItem label="Connect to the Proxy or Auth Service" options="Helm Chart">
```

Comment:
```suggestion
<TabItem label="Connect to your Teleport Cluster" options="Helm Chart">
```
To make this more general, since in the "Helm Chart" option you'd always be connecting to the Auth Service through the Proxy Service.